blockchain/contracts/NFTtheWorld.sol

```diff
@@ -108,20 +117,29 @@ contract NFTtheWorld {
         for (uint256 i = 0; i < joinedUsers[_dropHash].length; i++) {
             for (
                 uint256 j = 0;
-                j < nftReservations[joinedUsers[_dropHash][i]];
+                j < nftReservations[joinedUsers[_dropHash][i]][_dropHash];
                 j++
             ) {
-                nftElement = generateRandomNumber(j) % availableNFTs[_dropHash].length;
+                nftElement =
+                    generateRandomNumber(j) %
+                    availableNFTs[_dropHash].length;
                 remove(nftElement, _dropHash);
-                nftOwnerships[_dropHash][nftElement].reservedFor = joinedUsers[_dropHash][i];
-                nftReservationInformationOfUsers[joinedUsers[_dropHash][i]].push(nftOwnerships[_dropHash][nftElement].nftId);
+                nftOwnerships[_dropHash][nftElement].reservedFor = joinedUsers[
+                    _dropHash
+                ][i];
+                nftReservationInformationOfUsers[joinedUsers[_dropHash][i]]
+                    .push(nftOwnerships[_dropHash][nftElement].nftId);
             }
         }
     }
 
     // This function lets a user buy her reserved NFTs (one at a time)
     //TODO: Think about a time span during which the reserved NFTs have to be bought
-    function buyNFT(uint256 _price, uint256 _nftHash, uint256 _dropHash) public payable {
+    function buyNFT(
+        uint256 _price,
+        uint256 _nftHash,
+        uint256 _dropHash
```

Comment:
Just curious, why do you use underscore variables?

Reply:
It's a common practice to use underscore for the function parameters in solidity :)